src/support/generation.c

```diff
@@ -188,6 +188,8 @@ __wt_gen_drain(WT_SESSION_IMPL *session, int which, uint64_t generation)
                     }
                     verbose_timeout_flags = true;
                 }
+                WT_ASSERT(session,
```

Comment:
Let's say the timeout is configured to be less than a minute, we should probably log a message before actually aborting in diagnostic mode. Thoughts?

Reply:
I agree - it would be good if there was always a message logged prior to (or along with) this generating an error.

Reply:
Added a message here 6bd5ff27e1d10b5e9b14528e8039feeb3248698d